zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -734,6 +734,34 @@ impl<F: Field> ExecutionConfig<F> {
                 let mut offset = 0;
 
                 self.q_step_first.enable(&mut region, offset)?;
+                // handle empty block conditions
+                if block.txs.is_empty() {
+                    // if enable padding to fix length in the future, just change `end_row` to
+                    // target length
+                    let end_row = 2;
```

Comment:
This variable name sounds a bit strange to me, because it makes me think that the last row is 2.  I think referencing the length / number of rows in the variable would be more clear.  Like `num_rows`

Reply:
sound good to me , will update it !

Reply:
updated in [4097a9d](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/713/commits/4097a9d59af9ab46a1f33ec508ab2923d52ffc12)